src/app/components/SocialLinks/index.tsx

```diff
@@ -92,36 +102,82 @@ const SocialLinks = ({
   title,
   summaries = [],
   id = 'social-links-1',
+  eventTrackingData = { componentName: 'social-links' },
 }: SocialLinksProps) => {
+  const viewTracker = useViewTracker(eventTrackingData);
+  const getClickTrackerHandler = useClickTrackerHandler;
   if (!summaries.length) {
     return null;
   }
 
   const hasMultipleItems = summaries.length > 1;
 
+  const buildPromoEventTrackingData = (promo: Summary, index: number) => {
+    const itemTracker = {
+      type: 'social-link-promo',
+      text: promo.title,
+      position: index + 1,
+      resourceId: promo.id,
+    };
+    return {
+      itemTracker,
+      ...eventTrackingData,
+    };
+  };
+
   return (
     <section
       role="region"
       aria-labelledby={id}
       data-testid={id}
       css={styles.container}
+      {...viewTracker}
     >
       <Heading level={2} id={id} css={styles.heading}>
         {title}
       </Heading>
       {hasMultipleItems ? (
         <ul css={styles.unorderedList} role="list">
-          {summaries.map(summary => {
+          {summaries.map((summary, index) => {
+            const promoEventTrackingData = buildPromoEventTrackingData(
+              summary,
+              index,
+            );
+            const clickTrackerHandler = getClickTrackerHandler(
+              promoEventTrackingData,
+            );
+            const hasDescription = Boolean(summary.description);
+
             return (
               <li css={styles.item} key={summary.title}>
-                <SocialLink summary={summary} />
+                <SocialLinkImage imageUrl={summary.imageUrl} />
+                <a
+                  href={summary.link}
+                  css={styles.link}
+                  {...clickTrackerHandler}
+                  {...(hasDescription && {
+                    'aria-labelledby': `social-link-label-${index}`,
```

Comment:
Instead of recreating the `SocialLink` component stuff again, we could just pass in the `clickTrackerHandler` as a prop, e.g.:
```
<li css={styles.item} key={summary.title}>
    <SocialLink
         summary={summary}
         clickTrackerHandler={clickTrackerHandler}
     />
</li>
```